internal/services/springcloud/spring_cloud_service_resource.go

```diff
@@ -270,6 +270,12 @@ func resourceSpringCloudService() *pluginsdk.Resource {
 				},
 			},
 
+			"zone_redundant": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  true,
```

Comment:
This is a breaking change since we don't pass this in now and it gets defaulted to false